index.js

```diff
@@ -5,6 +5,7 @@ const webpack = require('webpack');
 const pDefer = require('p-defer');
 
 const observeWebpackCompiler = require('./lib/observeWebpackCompiler');
+const standardFs = require('./lib/standardFs');
```

Comment:
Maybe change the module to nodeFs

Reply:
Also should we add that we are doing this in the README?

Reply:
Done